src/datasets.c

```diff
@@ -532,6 +533,62 @@ Dataset *DatasetGet(const char *name, enum DatasetTypes type,
     return NULL;
 }
 
+static bool DatasetIsStatic(Dataset *set)
+{
+    // A set is static if it does not have any dynamic properties like
+    // save and/or state defined.
+    if (set->save == NULL || strcmp(set->save, "") == 0) {
```

Comment:
I would like to add a condition: that load *is* set. It is possible to have a 'dynamic' set that just doesn't save the data its tracking.